package.json

```diff
@@ -310,11 +310,11 @@
     "react-tagsinput": "^3.20.3",
     "redux": "^5.0.1",
     "redux-thunk": "^3.1.0",
-    "remark-gfm": "^1.0.0",
+    "remark-gfm": "^4.0.0",
     "resizable-panels-react": "^2.0.8",
```

Comment:
Issue around `remark-gfm`

Reply:
https://github.com/StatTag/StatWrap/assets/96688318/546bfca7-426e-43f8-bdd8-ef40f88cc3c2

Video demo for remark-gfm package older version, it used to function similarly. The strikethrough in this case is only enabled if there are equal no. of '~' on both sides

Reply:
But the errors are not related solely to strikethrough functionality. I tested it with the features listed here and encountered some errors (https://www.npmjs.com/package/remark-gfm), which were not present in version 1 but appeared in version 4.

Reply:
@Abhijay007 Got to know that the latest remark-gfm v4 only works with latest react-markdown , i.e., v9, so, I'm downgrading remark-gfm to v3, till we don't upgrade react-markdown, along with other react packages, which have some breaking changes. v3 is still an upgrade from v1 anyways.

Reply:
Thanks @AdiAkhileshSingh15 for looking into this. Two more things I want to mention:

1. Please run `yarn upgrade-interactive --latest` for the latest results and upgrade the dependencies which have minor changes. I noticed you did this for most of them, but I believe there are a few more left.

2. Secondly, please squash your commits.

Reply:
v2 and v3 are bringing in some issues with missing modules, which we face while upgrading react-markdown, 'unified' module not foud.
So, I believe the gfm upgrade can be performed parallely with react-markdown, we have to keep this in mind.

Reply:
so it is better to downgrade it to version one for now